apps/api/src/app.ts

```diff
@@ -36,7 +36,7 @@ app.setErrorHandler(function (error, _request, reply) {
 });
 
 app.register(cookiePlugin);
-app.register(fastifyJwt, { secret: env.JWT_SECRET });
+app.register(fastifyJwt, { secret: env.JWT_SECRET as string });
```

Comment:
why do you have to use `as` here? `env` is already giving this type defined right?